tips/TIP-0042/tip-0042.md

```diff
@@ -0,0 +1,1204 @@
+---
+tip: 42
+title: Account Output Type
+description: Defines the IOTA 2.0 Account used to issue blocks onto the network and stake IOTA coins
+author:
+  Philipp Gackstatter (@PhilippGackstatter) <philipp.gackstatter@iota.org>, Andrew Cullen (@cyberphysic4l)
+  <andrew.cullen@iota.org>
+discussions-to: TODO
+status: Draft
+type: Standards
+layer: Core
+created: 2023-05-03
+requires: TIP-21, TIP-22, TIP-38, TIP-45 and TIP-47
+replaces: TIP-18
+---
+
+# Summary
+
+The account is the central component of the IOTA 2.0 ledger that enables:
+
+- Issuance of blocks by burning Mana.
+- Staking IOTA coins for the validation of the network.
+- Secure ownership of all digital assets on the ledger.
+
+Its predecessor is the Alias Output, defined in [TIP-18](../TIP-0018/tip-0018.md). However, its functionality is
+effectively split between the Account Output and the [Anchor Output](../TIP-0054/tip-0054.md).
+
+## Summary of changes compared to TIP-18
+
+- Add `Mana` field to the `Account`.
+- Remove `Native Tokens` field. See [TIP-38 (Native Token Migration)](../TIP-0038/tip-0038.md#native-token-migration)
+  for migration details.
+- Add _Block Issuer Feature_.
+  - Mana owned by an account with a _Block Issuer Feature_ is bound to that account and can only be moved out with a
+    delay.
```

Comment:
What is the `delay` here?

Reply:
It's what is described in the Block Issuer Feature's [Additional Transaction Semantic Validation Rules](https://github.com/iotaledger/tips/blob/tip42/tips/TIP-0042/tip-0042.md#additional-transaction-semantic-validation-rules). You can put mana from an account (potential mana generated by it or its stored mana) into a Basic Output with a sufficiently long Timelock UC to move it out. Once that expires, you can move it anywhere. In practice this only takes a couple of minutes to unlock.

Reply:
Thanks, maybe expanding on that in this section would also help others to better understand it.
Maybe something like this:
`and can only be moved out with a delay by moving it into a Basic Output owned by that account with a sufficiently long Timelock UC to move it out. Once that expires, the Mana can be moved anywhere.`

Reply:
Yes, good point. Although I now updated the Block Issuer Feature section, since this section here is the summary, which should be kept very concise in my opinion.